gumroad/README.txt

```diff
@@ -19,11 +19,11 @@ For more information on how this plugin works, you can see a video on our Overla
 
 You can also use shortcodes:
 
-Basic overlay example: `[gumroad id="DviQY"]`
+Basic overlay example: `[gumroad id="DviQY" type="overlay"]`
 
 Basic embed example: `[gumroad id="GAPdj" type="embed"]`
 
-Overlay that will automatically show the payment form: `[gumroad id="DviQY" text="Purchase Item" wanted="true"]`
+Overlay that will automatically show the payment form: `[gumroad id="DviQY" text="Purchase Item" wanted="true" type="overlay"]`
```

Comment:
these can be moved to a diff pr since it is independent of gutenberg block support